scripts/build_lbann.sh

```diff
@@ -48,7 +48,7 @@ CMD_LINE_VARIANTS=
 
 # Default versions of Hydrogen, DiHydrogen, and Aluminum - use head of repo
 HYDROGEN_VER="@develop"
-ALUMINUM_VER="@1.3.1:"
+ALUMINUM_VER="@master"
```

Comment:
we may want to release a version here. OK for now

Reply:
Waiting for LLNL/Aluminum#208 to merge before doing another release.